src/cpu/arm.rs

```diff
@@ -266,10 +245,28 @@ prefixed_export! {
     static mut OPENSSL_armcap_P: u32 = ARMCAP_STATIC;
 }
 
-const _APPLE_TARGETS_HAVE_ALL_FEATURES: () = assert!(
-    (ARMCAP_STATIC == _ALL_FEATURES_MASK)
-        || !cfg!(all(target_arch = "aarch64", target_vendor = "apple"))
-);
+// MSRV: Enforce 1.61.0 on some aarch64-* targets (aarch64-apple-*, in particular) prior to. Earlier
+// versions of Rust before did not report the AAarch64 CPU features correctly for these targets.
+// Cargo.toml specifies `rust-version` but versions before Rust 1.56 don't know about it.
```

Comment:
I verified that `cargo test +1.60.0 --target=aarch64-apple-darwin` fails due to this, and `cargo test +1.61.0 --target=aarch64-apple-darwin` passes.